src/pages/help-centre/_article.tsx

```diff
@@ -36,6 +36,7 @@ const TabWrapper = styled.div`
 const ContactContainer = styled.div`
     margin-top: 8rem;
 `
+const platform = queryParamData() ? queryParamData() : ''
```

Comment:
Isn't this handled by the else case in `queryParamData`?

Reply:
That's right will remove the condition, Thanks Sean.